app/models/patient.rb

```diff
@@ -227,6 +227,8 @@ def discard_data
     phone_numbers.discard_all
     prescription_drugs.discard_all
     teleconsultations.discard_all
+    notifications.discard_all
+    treatment_group_memberships.discard_all
```

Comment:
Feels to me like we should `delete` or `destroy` these records rather than `discard`.  There isn't much reason to soft delete a notification record.